### PR TITLE
[MRG] Add a version.py => 2.0 alpha, add khmer@master to requirements.txt

### PR DESCRIPTION
**Pull request by ctb**
Relies on [sourmash_lib/VERSION](https://stackoverflow.com/questions/2058802/how-can-i-get-the-version-defined-in-setup-py-setuptools-in-my-package) to read the version from a single location.

- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make coverage` Is the new code covered?
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by ctb** [comment]
Fixes #98 and #110.

**Comment by ctb** [comment]
Ready for review & maybe merge.

File: setup.py [hunk 1]
```diff
@@ -4,7 +4,9 @@
 from setuptools import Extension
 import os
 
-VERSION="1.1"
+VERSION='undefined'
+with open('sourmash_lib/version.py') as f:
+    exec(f.read())
```
Comment (luizirber) [inline setup.py]: Hmmmm, can we use the other alternative from that answer, the one that doesn't need `exec()`?

Comment (ctb) [inline setup.py]: the pkg_resources one?

Comment (luizirber) [inline setup.py]: the "Store version string for use during install" one

**Comment by ctb** [comment]
Ready for review!

**Pull request merged**